install/ontology/ltiroles_membership.rdf

```diff
@@ -50,4 +50,12 @@
     <generis:includesRole rdf:resource="http://www.tao.lu/Ontologies/TAOLTI.rdf#LtiBaseRole"/>
     <generis:isSystem rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#True"/>
   </rdf:Description>
+  <rdf:Description rdf:about="http://purl.imsglobal.org/vocab/lis/v2/membership#Learner">
```

Comment:
We need a migration with syncModels() for this change to take effect on existing instances

Reply:
We have one in ltiDeliveryProvider, should I duplicate it?